cli/pkg/local/app.go

```diff
@@ -244,6 +244,8 @@ func (a *App) Reconcile() error {
 	}
 	if len(res.Errors) == 0 {
 		a.Logger.Infof("Hydration completed!")
+	} else if strict {
+		a.Logger.Fatalf("Hydration failed with errors:%v", res.Errors)
```

Comment:
Just do `a.Logger.Fatalf("Hydration failed")` here – the build errors are emitted earlier in the logs